{
    "dataType": "CVE_RECORD",
    "dataVersion": "5.1",
    "cveMetadata": {
        "state": "PUBLISHED",
        "cveId": "CVE-2022-3576",
        "assignerOrgId": "db201096-a0cc-46c7-9a55-61d9e221bf01",
        "assignerShortName": "synology",
        "datePublished": "2022-10-20T05:50:24.922383Z",
        "dateUpdated": "2024-09-16T19:36:43.491Z",
        "dateReserved": "2022-10-18T00:00:00"
    },
    "containers": {
        "cna": {
            "datePublic": "2022-10-20T00:00:00",
            "providerMetadata": {
                "orgId": "db201096-a0cc-46c7-9a55-61d9e221bf01",
                "shortName": "synology",
                "dateUpdated": "2022-10-20T00:00:00"
            },
            "descriptions": [
                {
                    "lang": "en",
                    "value": "A vulnerability regarding out-of-bounds read is found in the session processing functionality of Out-of-Band (OOB) Management. This allows remote attackers to obtain sensitive information via unspecified vectors. The following models with Synology DiskStation Manager (DSM) versions before 7.1.1-42962-2 may be affected: DS3622xs+, FS3410, and HD6500."
                }
            ],
            "affected": [
                {
                    "vendor": "Synology",
                    "product": "DiskStation Manager (DSM)",
                    "versions": [
                        {
                            "version": "unspecified",
                            "lessThan": "7.1.1-42962-2",
                            "status": "affected",
                            "versionType": "custom"
                        }
                    ]
                }
            ],
            "references": [
                {
                    "url": "https://www.synology.com/security/advisory/Synology_SA_22_17"
                }
            ],
            "metrics": [
                {
                    "cvssV3_1": {
                        "version": "3.1",
                        "vectorString": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:L/I:N/A:N",
                        "attackVector": "NETWORK",
                        "attackComplexity": "LOW",
                        "privilegesRequired": "NONE",
                        "userInteraction": "NONE",
                        "scope": "UNCHANGED",
                        "confidentialityImpact": "LOW",
                        "integrityImpact": "NONE",
                        "availabilityImpact": "NONE",
                        "baseScore": 5.3,
                        "baseSeverity": "MEDIUM"
                    }
                }
            ],
            "problemTypes": [
                {
                    "descriptions": [
                        {
                            "type": "CWE",
                            "lang": "en",
                            "description": "CWE-125: Out-of-bounds Read",
                            "cweId": "CWE-125"
                        }
                    ]
                }
            ]
        },
        "adp": [
            {
                "providerMetadata": {
                    "orgId": "af854a3a-2127-422b-91ae-364da2661108",
                    "shortName": "CVE",
                    "dateUpdated": "2024-08-03T01:14:02.548Z"
                },
                "title": "CVE Program Container",
                "references": [
                    {
                        "url": "https://www.synology.com/security/advisory/Synology_SA_22_17",
                        "tags": [
                            "x_transferred"
                        ]
                    }
                ]
            }
        ]
    }
}